webserver/compose.yaml

```
@@ -46,6 +46,7 @@ services:
       REDIS_DATABASE: 0
       REDIS_PASSWORD: ""
     volumes:
+    - webserver-uploads-images:/app/public/uploads/images
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)